deployments/helm/KubeArmorOperator/values.yaml

```diff
@@ -4,3 +4,26 @@ kubearmorOperator:
     repository: kubearmor/kubearmor-operator
     tag: latest
   imagePullPolicy: IfNotPresent
+
+  configSpec:
+    defaultCapabilitiesPosture: audit
+    defaultFilePosture: audit
+    defaultNetworkPosture: audit
+
+    defaultVisibility: process,file,network
+
+    kubearmorImage:
+        image: kubearmor/kubearmor:stable
+        imagePullPolicy: Always
+
+    kubearmorInitImage:
+        image: kubearmor/kubearmor-init:stable
```

Comment:
shouldn't we be moving to v0.11.0 instead on stable tag?

Reply:
Currently, we are following the flow of using `stable` and `latest` releases. If we decide to change that in future, then I think we should handle that in a separate PR.